### PR TITLE
pybind/mgr: Fix Python 3 imports in diskprediction & insights

### PR DESCRIPTION
**Pull request by sebastian-philipp**
Signed-off-by: Sebastian Wagner <sebastian.wagner@suse.com>

### CODE REVIEW

File: src/pybind/mgr/diskprediction/common/__init__.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from __future__ import absolute_import
 import errno
 from functools import wraps
-from httplib import BAD_REQUEST
+from six.moves.http_client import BAD_REQUEST
```
[the remark this answers is not in the document]
Comment (sebastian-philipp) [reply src/pybind/mgr/diskprediction/common/__init__.py]:
```
hexdump -C __init__.py | head | tail -n 5
00000050  73 0d 0a 66 72 6f 6d 20  73 69 78 2e 6d 6f 76 65  |s..from six.move|
00000060  73 2e 68 74 74 70 5f 63  6c 69 65 6e 74 20 69 6d  |s.http_client im|
00000070  70 6f 72 74 20 42 41 44  5f 52 45 51 55 45 53 54  |port BAD_REQUEST|
00000080  0d 0a 69 6d 70 6f 72 74  20 6f 73 0d 0a 69 6d 70  |..import os..imp|
00000090  6f 72 74 20 73 69 67 6e  61 6c 0d 0a 0d 0a 0d 0a  |ort signal......|
```
`0d 0a` looks innocent to me ?

[the remark this answers is not in the document]
Comment (tserong) [reply src/pybind/mgr/diskprediction/common/__init__.py]: It does indeed. Probably won't actually break anything, I just thought it was a bit odd.

**Comment by noahdesu** [comment]
this worked locally for me

File: src/pybind/mgr/diskprediction/common/__init__.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (noahdesu) [reply src/pybind/mgr/diskprediction/common/__init__.py]: some carriage returns have managed to slip through according to a quick grep, but i generally think they should be removed if noticed.

**Comment by smithfarm** [comment]
@yuriw @tchaikov @jcsp Could this be included in an upcoming integration run?

**Comment by tchaikov** [comment]
@smithfarm yeah, and better off running it with `qa/suites/rados/mgr/tasks/module_selftest.yaml`.

**Pull request merged**